pkgs/development/compilers/ghc/8.2.2.nix

```diff
@@ -17,6 +18,9 @@
   # library instead of the faster but GPLed integer-gmp library.
   enableIntegerSimple ? !(stdenv.lib.any (stdenv.lib.meta.platformMatch stdenv.hostPlatform) gmp.meta.platforms), gmp
 
+, # Allows this GHC to produce binaries that have DWARF debug symbols.
+  enableDwarf ? (!stdenv.isDarwin) # elfutils libdw supports only ELF
```

Comment:
Is there a better expression than `!stdenv.isDarwin` I could use that gives me "all ELF platforms"?

Reply:
@angerman on `#ghc` chat said:

> I doubt there is any consumer of Mach-o that is not macOS or iOS or really exotic. Same for Windows. Everyone else just uses elf.